src/components/main-navigation/script.ts

```diff
@@ -387,5 +391,14 @@ Polymer({
     if (a.index > b.index) order = 1;
 
     return order;
+  },
+  documentOnClick: function (event) {
+    var navItems = this.querySelectorAll('nav-item > a');

```

Comment:
Need to be tested for other applications.
Im affraid this will cause issues for applications that works with the current solution (not using requirejs)